docs/content-flow.md

```diff
@@ -0,0 +1,388 @@
+
+### Image Format
+
+Images in a registry normally are stored in the following format. An "image" is actually just a json document
+that lists either the content that will combine to provide to make that image, or an index of other such documents
+that vary by platform.
+
+Here is the document for our example image, `redis:5.0.9`:
+
+```json
+{
+  "manifests": [
+    {
+      "digest": "sha256:a5aae2581826d13e906ff5c961d4c2817a9b96c334fd97b072d976990384156a",
+      "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+      "platform": {
+        "architecture": "amd64",
+        "os": "linux"
+      },
+      "size": 1572
+    },
+    {
+      "digest": "sha256:4ff8940144391ecd5e1632d0c427d95f4a8d2bb4a72b7e3898733352350d9ab3",
+      "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+      "platform": {
+        "architecture": "arm",
+        "os": "linux",
+        "variant": "v5"
+      },
+      "size": 1573
+    },
+    {
+      "digest": "sha256:ce541c3e2570b5a05d40e7fc01f87fc1222a701c81f95e7e6f2ef6df1c6e25e7",
+      "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+      "platform": {
+        "architecture": "arm",
+        "os": "linux",
+        "variant": "v7"
+      },
+      "size": 1573
+    },
+    {
+      "digest": "sha256:535ee258100feeeb525d4793c16c7e58147c105231d7d05ffc9c84b56750f233",
+      "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+      "platform": {
+        "architecture": "arm64",
+        "os": "linux",
+        "variant": "v8"
+      },
+      "size": 1573
+    },
+    {
+      "digest": "sha256:0f3b047f2789547c58634ce88d71c7856999b2afc8b859b7adb5657043984b26",
+      "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+      "platform": {
+        "architecture": "386",
+        "os": "linux"
+      },
+      "size": 1572
+    },
+    {
+      "digest": "sha256:bfc45f499a9393aef091057f3d067ff7129ae9fb30d9f31054bafe96ca30b8d6",
+      "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+      "platform": {
+        "architecture": "mips64le",
+        "os": "linux"
+      },
+      "size": 1572
+    },
+    {
+      "digest": "sha256:3198e1f1707d977939154a57918d360a172c575bddeac875cb26ca6f4d30dc1c",
+      "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+      "platform": {
+        "architecture": "ppc64le",
+        "os": "linux"
+      },
+      "size": 1573
+    },
+    {
+      "digest": "sha256:24a15cc9366e1557db079a987e63b98a5abf4dee4356a096442f53ddc8b9c7e9",
+      "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
+      "platform": {
+        "architecture": "s390x",
+        "os": "linux"
+      },
+      "size": 1573
+    }
+  ],
+  "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
+  "schemaVersion": 2
+}
+```
+
+As you can see, the `mediaType` gives the type as a "manifest list", or in OCI parlance, an index. It contains
```

Comment:
Instead of "as you can see", "the json document shows the 'mediaType' is a 'manifest list'..."

Reply:
Please note I'm not a professional doc writer, so I don't know if this advice eschews good patterns for keeping the reader engaged here...
Really just wanting to not assume the reader can see this.

Reply:
Updated these as well.

Reply:
> I'm not a professional doc writer

Neither am I! :-) Although I do write an immense amount.

The hardest part of writing (and giving directions) is putting yourself in the shoes of a reader, i.e. not falling prey to assuming they know what you know. I am very happy to take any suggestions that help with that.